container/Home/Search/SearchForm.js

```diff
@@ -134,9 +92,30 @@ const SearchForm = () => {
     );
   };
 
+  const handlerOperationType = (value) => {
+    setOperationType(value)
+  }
+  const handlerPropertyOption = value => {
+    setPropertyOption(value)
+  }
+  const { Option } = Select
+
   return (
-    <FormWrapper>
-      <ComponentWrapper>
+    <FormWrapperHome>
+      <ComponentWrapper className='operation-select'>
+        <Select 
+          defaultValue='Alquiler'
+          onChange={handlerOperationType}
+          bordered={false}
+          suffixIcon={<FaAngleDown className='operation-marker'/>}
+          className='selectbox'
+          >
+          <Option value='alquiler'>Alquiler</Option>
```

Comment:
save this labels and values on a constant: `const OPTIONS = [{ label: 'Alquiler', value: 'rental'  }, ...];` and the default value as `defaultValue={OPTIONS[0].value}`.